src/ui/components/Library/BatchActionDropdown.tsx

```diff
@@ -105,6 +119,12 @@ function BatchActionDropdown({
       distance={0}
     >
       <Dropdown menuItemsClassName="z-50">
+        {isPublicEnabled ? (
+          <>
+            <DropdownItem onClick={() => setSelectedIdsIsPrivate(true)}>Make private</DropdownItem>
+            <DropdownItem onClick={() => setSelectedIdsIsPrivate(false)}>Make public</DropdownItem>
```

Comment:
If we were feeling fancy 😁 we could disable either of these options when all of the selected IDs were already private or public

Reply:
Ah yeah, I'll do this soon since our current `DropdownItem` implementation doesn't accept `disabled`. Our `DropdownButton` component does, but doesn't accept `onClick` 😅